drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -36,6 +36,9 @@ def self.configuration
     description 'HUD Client'
     field :id, ID, null: false
     field :lock_version, Integer, null: false
+    field :assessment_eligibilities, [HmisSchema::AssessmentEligibility], null: false do
+      argument :enrollment_id, ID, required: true
+    end
```

Comment:
This could also go on enrollment. Having this on client leaves open the possibility that we might not have an enrollment but want to query assessments are applicable for a given client/project combination (perhaps multiple intakes?).

Reply:
Thanks for the feedback! Sounds like this may be too much of a departure from the existing pattern, particularly if there's no reasonable use-case we can anticipate were we would need this flexibility.

I can move this field to enrollment assuming there will never be more than one type of intake for a given project. For example when enrolling a client, there will never be a need for the user to choose between different customized intakes. A use case I could imagine would be a project wanting to pilot or demo new intake forms.

Reply:
I think it's fair to assume there will be 1 intake assessment.

If a community wanted to pilot a modified intake assessment, they would do something like:
* create a new FormDefinition called "modified intake assessment", with role INTAKE
* create a form instance specifying that modified intake assessment should be used for heads of households in PSH programs (for example)
* that would lead to certain clients getting different intake assessments than other clients. I don't think we need to support letting the user choose which intake. something like that could be supported with custom assessments if we really need it.

Reply:
So yeah, lets switch it over to enrollment then. Thanks!